src/components/cc-zone-picker/cc-zone-picker.js

```diff
@@ -0,0 +1,174 @@
+          <legend>
+            <cc-icon class="zone-legend-icon" .icon="${zoneIcon}" size="lg"></cc-icon>
+            <span class="zone-legend-text"> ${i18n('cc-zone-picker.legend')} </span>
+          </legend>
+          <div class="zone-section">
+            ${this.zonesSection.map(
+              (zoneSection) => html`
+                ${!isStringEmpty(zoneSection.title)
+                  ? html` <div class="zone-section-title">${zoneSection.title}</div> `
+                  : ''}
+                <div class="form-controls">
+                  ${zoneSection.zones.map((zone) => this._renderZoneCard(zone, this.value))}
+                </div>
+              `,
+            )}
+          </div>
+        </fieldset>
+      </form>
+    `;
+  }
+
+  /**
+   * @param {ZoneItem} zone
+   * @param {string} selectedZoneCode
+   */
+  _renderZoneCard(zone, selectedZoneCode) {
+    const isSelected = zone.code === selectedZoneCode;
+    return html`
+      <div>
+        <input
+          class="visually-hidden"
+          type="radio"
+          name="zone"
+          .value="${zone.code}"
+          ?disabled=${zone.disabled}
+          ?checked="${isSelected}"
+          id="${zone.code}"
+        />
+        <label for="${zone.code}">
+          <cc-zone-card
+            ?selected="${isSelected}"
+            ?disabled="${zone.disabled}"
+            name="${zone.name}"
+            country="${zone.country}"
+            code="${zone.code}"
+            flagUrl="${zone.flagUrl}"
+            .images="${zone.images}"
+          ></cc-zone-card>
+        </label>
+      </div>
+    `;
+  }
+
+  static get styles() {
+    return [
+      accessibilityStyles,
+      // language=CSS
+      css`
+        :host {
+          display: block;
+        }
+
+        legend {
+          display: flex;
+          gap: 0.25em;
+        }
+
+        .form-controls {
+          display: grid;
+          gap: 1em;
+          grid-template-columns: repeat(auto-fill, minmax(12.5em, 1fr));
+          margin: 1em 0 0 2em;
+        }
+
+        .zone-legend-icon {
+          --cc-icon-color: var(--cc-color-text-primary);
+
+          align-self: center;
+        }
+
+        .zone-legend-text {
+          color: var(--cc-color-text-primary-strongest);
+          font-family: var(--cc-ff-form-legend), inherit;
+          font-size: 1.625em;
+          font-weight: 500;
+        }
+
+        .zone-section-title {
+          color: var(--cc-color-text-primary-strongest);
+          font-family: var(--cc-ff-form-legend), inherit;
+          font-size: 1.15em;
+          margin: 1em 1.85em;
+        }
+
+        .private-zones,
+        .public-zones {
+          margin: 1em 0;
+        }
+
+        fieldset {
+          border: none;
+          margin: 0;
+          padding: 0;
+        }
+
+        cc-zone-card {
+          height: 100%;
+        }
+
+        input[type='radio']:focus-visible + label cc-zone-card {
+          border-radius: 2px;
```

Comment:
**question:** I can't remember why we need the `border-radius` here and why its value is not a token :thinking: